shared/chat/inbox/row/container.js

```diff
@@ -146,7 +146,8 @@ const ConnectedRow = compose(
   // $FlowIssue
   pausableConnect(mapStateToProps, mapDispatchToProps, mergeProps),
   branch(props => props.teamname && !props.channelname, renderComponent(TeamRow)),
-  branch(props => props.teamname && props.channelname, renderComponent(ChannelRow))
+  branch(props => props.teamname && props.channelname, renderComponent(ChannelRow)),
+  branch(props => props.filtered, renderComponent(FilteredRow))
```

Comment:
I noticed that flow doesn't catch, e.g. mistakenly putting `props.filter` or something -- will this be fixed once we update to 0.53? Is that what the $FlowIssue tag above is?